python/tvm/auto_scheduler/cost_model/xgb_model.py

```diff
@@ -141,6 +147,12 @@ def update(self, inputs, results):
         self.inputs.extend(inputs)
         self.results.extend(results)
 
+        if len(self.inputs) - self.last_train_length < self.last_train_length / 5:
```

Comment:
Add a new bool argument `adapative_training` as the switch of this feature.

Reply:
Added.